### ISSUE TITLE
fit stops after Iteration 0 when metric is r2

### ISSUE THREAD

**Issue by PardeepRassani** [microsoft/FLAML#104]
I am using FLAML in Django views: 

               X_train, X_test, y_train, y_test = train_test_split(df.copy(), train_size=selectedTrainingPercentage) 
               
               automl = AutoML()

                settings = {
                    "time_budget": 60,      # total running time in seconds
                    "metric": 'r2',         # primary metrics for regression can be chosen from: ['mae','mse','r2']
                                            # list of ML learners; we tune xgboost in this example
                    "task": 'regression',   # task type
                }
                print('fitting')

                automl.fit(X_train=X_train, y_train=y_train, **settings)
                print('fit complete')`

And the fitting stops at iteration 0:

![Screenshot from 2021-06-08 21-14-11](https://user-images.githubusercontent.com/36777973/121222167-bb243680-c89f-11eb-943c-1772eea72ef1.png)

However it works completely fine if I change the metric to nae or mse rather than r2.

**Comment by PardeepRassani** [comment]
There is no error message. Above screenshot is the complete output from flaml.

**Comment by sonichi** [comment]
That means the first training doesn't finish. Does the basic regression example work for you?

```python
from flaml import AutoML
from sklearn.datasets import load_boston
# Initialize an AutoML instance
automl = AutoML()
# Specify automl goal and constraint
automl_settings = {
    "time_budget": 10,  # in seconds
    "metric": 'r2',
    "task": 'regression',
    "log_file_name": "test/boston.log",
}
X_train, y_train = load_boston(return_X_y=True)
# Train with labeled input data
automl.fit(X_train=X_train, y_train=y_train,
                        **automl_settings)
```

**Comment by PardeepRassani** [comment]
@sonichi  Yes, successfully.

**Comment by sonichi** [comment]
How can we reproduce this problem? We can chat in [gitter](https://gitter.im/FLAMLer/community) if you'd like to.

**Comment by sonichi** [comment]
closing as unable to reproduce.
